mdx-components.tsx

```diff
@@ -52,6 +51,7 @@ export function useMDXComponents(components: MDXComponents): MDXComponents {
     InlineFilter,
     MigrationAlert,
     YoutubeEmbed,
-    Banner
+    Banner,
+    ...components
```

Comment:
Can we please remove this , I this was added by mistake by me  `...components` build will fail and could be a `blocking change` in the pr